Stripe/STPAPIClient.m

```diff
@@ -69,14 +69,14 @@
 @implementation Stripe
 
 static NSArray<PKPaymentNetwork> *_additionalEnabledApplePayNetworks;
+static NSString *_defaultPublishableKey;
 
 + (void)setDefaultPublishableKey:(NSString *)publishableKey {
-    [STPAPIClient validateKey:publishableKey];
-    [STPPaymentConfiguration sharedConfiguration].publishableKey = publishableKey;
+    _defaultPublishableKey = [publishableKey copy];
 }
 
 + (NSString *)defaultPublishableKey {
-    return [STPPaymentConfiguration sharedConfiguration].publishableKey;
+    return _defaultPublishableKey;
```

Comment:
I might actually go so far as to make this non-nil (initial value of empty string) so that we can have true `null_resettable` behavior in STPAPIClient. Thoughts @davidme-stripe ?

Reply:
It's a bit weird to have that as an explicit default value, because an empty string isn't a valid value. Like, we fire an assertion if you try to set publishableKey to an empty string.